script/release/notes/index.ts

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env node
 
-const { GitProcess } = require('dugite');
-const minimist = require('minimist');
-const path = require('node:path');
-const semver = require('semver');
+import { GitProcess } from 'dugite';
+import { basename } from 'node:path';
+import { valid, compare, gte, lte } from 'semver';
 
-const { ELECTRON_DIR } = require('../../lib/utils');
-const notesGenerator = require('./notes.js');
+import { ELECTRON_DIR } from '../../lib/utils';
+import { get, render } from './notes';
 
-const { Octokit } = require('@octokit/rest');
-const { createGitHubTokenStrategy } = require('../github-token');
+import { Octokit } from '@octokit/rest';
+import { createGitHubTokenStrategy } from '../github-token';
+import { parseArgs } from 'node:util';
```

Comment:
Newline between imports and `const octokit`. Would also be nice to put the `node:util` import up with the first block.

Reply:
There are issues like this all over the place, I fixed a few but want to follow up with a import ordering / grouping lint rule